docs/overview.rst

```diff
@@ -0,0 +1,56 @@
+PIE Service Overview
```

Comment:
@alangsto This is great! Really concise and also clear. I feel like you provided just the right amount of information for a user too.